acl/src/main/java/org/apache/rocketmq/acl/common/Permission.java

```diff
@@ -26,7 +26,7 @@
 public class Permission {
 
     public static final byte DENY = 1;
-    public static final byte ANY = 1 << 1;
+    public static final byte ANY = 0b00001110;
```

Comment:
What's the reason to change the value of ANY?

Reply:
Keep the original value and the judgment will fail
<img width="555" alt="image" src="https://user-images.githubusercontent.com/31796988/221735061-2a68edfd-c390-4b31-987d-06ec905b04d9.png">

Reply:
I notice that the semantics of ANY don't seem to be SUB+PUB